benchmark/Streamly/Benchmark/Data/Parser.hs

```diff
@@ -628,6 +630,27 @@ choice value =
         (PR.choice (replicate value (PR.satisfy (< 0))) AP.<|> PR.satisfy (> 0))
 -}
 
+{-# INLINE sourceUnfoldrMc #-}
+sourceUnfoldrMc :: Monad m => Int -> Int -> Stream m Char
```

Comment:
Can the suffix `c` be capital?
Also why do we need it?
Can't we just use `sourceUnfoldrM`? and map `chr` on it?